lib/main.dart

```diff
@@ -11,7 +11,57 @@ class RationCalculatorApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
-      home: CowCharacteristicsPage(),
+      theme: ThemeData(
+        useMaterial3: true,
+        colorScheme: ColorScheme.fromSeed(seedColor: Colors.green),
+      ),
+      home: HomePage(),
+    );
+  }
+}
+
+class HomePage extends StatefulWidget {
+  const HomePage({super.key});
+
+  @override
+  State<HomePage> createState() => _HomePageState();
+}
+
+class _HomePageState extends State<HomePage> {
+  int _selectedIndex = 0;
+
+  final List<Widget> _pages = [
+    CowCharacteristicsPage(),
+    Placeholder(),
```

Comment:
The place holder will just be replaced with a link to the FeedingGuidelinesPage from:
https://github.com/minerva-university/rationapp/blob/main/lib/screens/feeding_guidelines_page.dart

Reply:
I pulled this branch locally, and made the change suggested above. There's good news and bad news. The good news is that the page renders just fine. The bad news is that it also breaks navigation.  I think we should take the approach mentioned in the other comment.

Reply:
yeah, I agree the toggle will quickly become complicated with new pages
the last demo in the NaviagtionBar docs is exactly what I was looking for. I'll implement the changes.